pymc3/distributions/multivariate.py

```diff
@@ -998,6 +1013,90 @@ def logp(self, x):
 
         return norm + logp_lkj + logp_sd + det_invjac
 
+    def _random(self, n, eta, size=1):
+        eta_sample_shape = (size,) + eta.shape
```

Comment:
Since there are large overlap with the LKJCorr random implementation, we should isolate the function outside of the class so both LKJCorr and LKJCorrCholesky can call it.

Reply:
@junpenglao any update?

Reply:
@lucianopaz @twiecki Yes the current implementation generate a Cholesky factor (`P` in the random generation code), we can then use the LDL decomposition https://en.wikipedia.org/wiki/Cholesky_decomposition#LDL_decomposition to generate the LKJCorrCholesky.